packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs

```diff
@@ -46,6 +46,10 @@
         	"occurrenceDateTime":"{{formatAsDateTime imm.effectiveTime.low.value}}",
         {{else if imm.effectiveTime.value}}
         	"occurrenceDateTime":"{{formatAsDateTime imm.effectiveTime.value}}",
+        {{else if @encounterTimePeriod.low}}
```

Comment:
why not use 
`            "onsetPeriod":{{>Utils/PeriodOrDefaultPeriod.hbs period=conditionEntry.effectiveTime}},
`

Reply:
Oh I see no period supported.

Reply:
I would put High before low in the order of the else if
Sometimes the low is the patients date of birth if the encounter date time is messed up

Reply:
Sounds good!